snap/snapcraft.yaml

```diff
@@ -21,26 +24,71 @@ apps:
       - optical-drive
       - removable-media
     common-id: dev.linwood.butterfly
-    desktop: usr/share/applications/dev.linwood.butterfly.desktop
+    desktop: meta/gui/butterfly.desktop
 
 parts:
+  flutter-git:
+    source: https://github.com/flutter/flutter.git
+    source-tag: 3.10.1
+    plugin: nil
```

Comment:
Do we need plugin: nil?

Reply:
I have mentioned the needs in the commit messages, please read them
>Using flutter git, and building from source, for better integrity and reduces the need of any file. In future when flutter stable will be updated, we can use the flutter plugin directly, without any flutter custom commands in override-build

Reply:
Oh okay, I hadn't seen these messages in the preview